2004-07-28  Marco Gerards  <metgerards@student.han.nl>

	* io-select.c: Include <hurd/ports.h>.
	(netfs_S_io_select): Add arguments `reply' and `replytype'.
	* mutations.h (IO_SELECT_REPLY_PORT): New macro.


Index: io-select.c
===================================================================
RCS file: /cvsroot/hurd/hurd/libnetfs/io-select.c,v
retrieving revision 1.2
diff -u -p -r1.2 io-select.c
--- libnetfs/io-select.c	9 May 1996 22:15:46 -0000	1.2
+++ libnetfs/io-select.c	28 Jul 2004 12:40:29 -0000
@@ -1,5 +1,5 @@
 /* 
-   Copyright (C) 1995, 1996 Free Software Foundation, Inc.
+   Copyright (C) 1995, 1996, 2004 Free Software Foundation, Inc.
    Written by Michael I. Bushnell, p/BSG.
 
    This file is part of the GNU Hurd.
@@ -20,9 +20,12 @@
 
 #include "netfs.h"
 #include "io_S.h"
+#include <hurd/ports.h>
 
 error_t
 netfs_S_io_select (struct protid *user,
+		   mach_port_t reply,
+		   mach_msg_type_name_t replytype,
 		   int *type)
 {
   if (!user)
Index: mutations.h
===================================================================
RCS file: /cvsroot/hurd/hurd/libnetfs/mutations.h,v
retrieving revision 1.3
diff -u -p -r1.3 mutations.h
--- libnetfs/mutations.h	14 Dec 1995 23:30:59 -0000	1.3
+++ libnetfs/mutations.h	28 Jul 2004 12:40:29 -0000
@@ -1,5 +1,5 @@
 /* 
-   Copyright (C) 1995 Free Software Foundation, Inc.
+   Copyright (C) 1995, 2004 Free Software Foundation, Inc.
    Written by Michael I. Bushnell, p/BSG.
 
    This file is part of the GNU Hurd.
@@ -20,6 +20,8 @@
 
 /* Only CPP macro definitions should go in this file. */
 
+#define IO_SELECT_REPLY_PORT
+
 #define FILE_INTRAN protid_t begin_using_protid_port (file_t)
 #define FILE_DESTRUCTOR end_using_protid_port (protid_t)
 

